pkg/api/payer/service.go

```
@@ -134,7 +136,18 @@
 				newClientEnvelopeWithIndex(i, clientEnvelope),
 			)
 		} else {
-			out.forNodes[aad.TargetOriginator] = append(out.forNodes[aad.TargetOriginator], newClientEnvelopeWithIndex(i, clientEnvelope))
+			// backwards compatibility
+			var targetNodeId uint32
+			if aad.GetTargetOriginator() != 0 {
```

golangci-lint (1.62.2)

SA1019: aad.GetTargetOriginator is deprecated: Marked as deprecated in xmtpv4/envelopes/envelopes.proto. 

(staticcheck)